### PR TITLE
1.17.1 cmr ref

### PR DESCRIPTION
**Pull request by fi3ld**

### CODE REVIEW

File: docs/command-reference.md [hunk 1]
```diff
@@ -4,794 +4,618 @@ title: Command reference
...
-```text title="Example for source NameNode HA cluster with Kerberos enabled"
-filesystem add hdfs --file-system-id mysource --source --default-fs hdfs://sourcenameservice --properties-files /etc/hadoop/conf/core-site.xml,/etc/hadoop/conf/hdfs-site.xml --kerberos-keytab /etc/security/keytabs/hdfs.headless.keytab --kerberos-principal hdfs@SOURCEREALM.COM
-```
 
-##### HDFS as target
 
-:::note
-When specifying a HDFS filesystem as a target, the property files for the target cluster must exist on the local filesystem and be accessible to the LiveData Migrator system user.
-:::
+### `help`
```
Comment (stepgerner) [inline docs/command-reference.md]: Sorry to inflict this pain on you, but the last version of the guide had ----s between every command to create a line: easily sectioning off each command and its parameters for readability. I think this was really helpful, and found the commands blur together a bit more without it. Can you return the horizontal lines?

File: docs/command-reference.md [hunk 2]
```diff
@@ -2262,291 +2194,523 @@ Add a remote agent to connect to a [Snowflake ](https://docs.snowflake.com/) war
...
+
+```text
+hive agent configure hive --name databricksAgent --access-token myexamplefg123456789t6fnew7dfdtoken4
+```
+
+### `hive agent configure dataproc`
+
+Change the configuration of an existing Dataproc agent using `hive agent configure dataproc`.
+
+The parameters that can be changed are the same as the ones listed in the [`hive agent add databroc`](#hive-agent-add-hive) section.
```
Comment (stepgerner) [inline docs/command-reference.md]: Typo here, "databroc" should be "hive", presumably

File: docs/command-reference.md [hunk 3]
```diff
@@ -4,794 +4,618 @@ title: Command reference
...
 
-Looking to learn more about LiveData Migrator commands? This reference page includes a comprehensive description of each command available from the LiveData Migrator CLI.
 
-Each command description includes the information available from the [`help`](#help) command. Tab-completion will also give you guidance when entering commands on the available options and help auto-complete the needed values.
+## System service commands
 
-:::tip
-You can also find information about UI configuration items here. Look for them in their equivalent command line interface (CLI) mandatory and optional parameters.
-:::
+The service scripts are used to control operation of each individual service.
```
[the remark this answers is not in the document]
Comment (stepgerner) [reply docs/command-reference.md:10]: Nevermind, resolved as per discussion

File: docs/command-reference.md [hunk 4]
```diff
@@ -4,794 +4,618 @@ title: Command reference
...
-        source fs show [--detailed]
-```
 
-#### Optional parameters
 
-* **`---detailed`** Include all configuration properties for the source filesystem in the response.
+<br/>
 
-## Filesystem commands
+## Connect to the LiveData Migrator CLI
```
[the remark this answers is not in the document]
Comment (stepgerner) [reply docs/command-reference.md:40]: Just any comment to explain why it's important/useful to connect to the CLI, and then state the below command is how to do it

File: docs/command-reference.md (hunk 2 above)
Comment (stepgerner) [reply docs/command-reference.md:2347]: Should be dataproc, link to dataproc

**Comment by fi3ld** [comment]
I removed these links due to bug. 

Error: Docusaurus found broken links!

Please check the pages of your site in the list bellow, and make sure you don't reference any path that does not exist.
Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.

Exhaustive list of all broken links found:

On source page path = /live-data-migrator/docs/command-reference:
-> linking to ./create-path-mappings.md (resolved as: /live-data-migrator/docs/create-path-mappings.md)
-> linking to ./configuration-hvm.md#tls-certificates (resolved as: /live-data-migrator/docs/configuration-hvm.md)
-> linking to ./configure-system-users.md#defaults (resolved as: /live-data-migrator/docs/configure-system-users.md)
-> linking to ./one-time-migration.md (resolved as: /live-data-migrator/docs/one-time-migration.md)
-> linking to ./troubleshooting.md#rule-names (resolved as: /live-data-migrator/docs/troubleshooting.md)
-> linking to ./manage-migrations.md#pending-regions (resolved as: /live-data-migrator/docs/manage-migrations.md)
-> linking to ./migration-verifications.md (resolved as: /live-data-migrator/docs/migration-verifications.md)
-> linking to ./migration-verifications.md#verification-status-values (resolved as: /live-data-migrator/docs/migration-verifications.md)

at Object.reportMessage (/Users/rob/wandisco_docs/wandisco-documentation-ldm/node_modules/@docusaurus/utils/lib/index.js:454:19)
at Object.handleBrokenLinks (/Users/rob/wandisco_docs/wandisco-documentation-ldm/node_modules/@docusaurus/core/lib/server/brokenLinks.js:142:17)
at async buildLocale (/Users/rob/wandisco_docs/wandisco-documentation-ldm/node_modules/@docusaurus/core/lib/commands/build.js:164:5)
at async tryToBuildLocale (/Users/rob/wandisco_docs/wandisco-documentation-ldm/node_modules/@docusaurus/core/lib/commands/build.js:33:28)
at async Object.mapAsyncSequencial (/Users/rob/wandisco_docs/wandisco-documentation-ldm/node_modules/@docusaurus/utils/lib/index.js:410:24)
at async build (/Users/rob/wandisco_docs/wandisco-documentation-ldm/node_modules/@docusaurus/core/lib/commands/build.js:71:25)

**Pull request merged**